texastribune/donations
# updates for stripe terminal functionality

#### What's this PR do?
Adds in functionality to allow for in-person donations through [stripe terminal](https://docs.stripe.com/terminal). The form for said in-person donations lives in the portal side of the app.

#### Why are we doing this? How does it help us?
We're trying to expand our options for collecting donations at live events, TribFest in particular.

#### How should this be manually tested?
* Visit the in-person donation form on [portal on testing](https://donations-testing.herokuapp.com/account/create-donation)
* Fill out whatever info you'd like to and click the Donate button
* You should see a success message pop up
    * because we are still in the testing phase, a pretend card reader swipe happens at this point
* Visit stripe, make sure you're in test mode and click through to the transactions
    * you should see the donation you added at the top of the list and it should be completed
* Visit salesforce and make sure the same donation has synced over by looking up the customer or subscription id

#### How should this change be communicated to end users?
We'll want to let the events and membership teams know when this is launched. Membership may have further thoughts and events might want to set up time to have help desk volunteers play with it ahead of 9/7.

#### Are there any smells or added technical debt to note?
There are a handful of things that will need to be handled when we have a physical card reader handy for doing more manual testing with.

#### What are the relevant tickets?
https://airtable.com/appyo1zuQd8f4hBVx/tbloNZu8GkM52NKFR/viwS1XPty68eK4Ett/rec62JcrUe2zDqbJF?blocks=hide
https://airtable.com/appyo1zuQd8f4hBVx/tbloNZu8GkM52NKFR/viwS1XPty68eK4Ett/recVVXUJpl0soHb2E?blocks=hide

#### Have you done the following, if applicable:
***(optional: add explanation between parentheses)***


#### TODOs / next steps:
